src/main/webapp/help-mavenInjectionDisabledNodes.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    The list of node labels where the Gradle Enterprise Maven Extension injection should be disabled.
+    By default, all nodes are enabled.
```

Comment:
same re: mentioning the precedence of disabled over enabled nodes

Reply:
Done.